frontend/src/newConsole/components/newLogos/LogosSidebar.tsx

```diff
@@ -37,26 +31,25 @@ export class LogosSideBarComponent extends React.PureComponent<Props, State> {
 
   public render() {
     const instance = this.props.activeInstance;
+    let logo;
+    let favicon;
+
+    if (instance.data && instance.data.logo) {
+      logo = instance.data.logo;
+    }
+    if (instance.data && instance.data.favicon) {
+      favicon = instance.data.favicon;
+    }
+
     return (
-      <ConsolePage
```

Comment:
This component is a page itself. So it should be wrapped in `ConsolePage`.

Reply:
Done like so